syntheseus/cli/search.py

```diff
@@ -46,7 +46,12 @@
 from syntheseus.search.mol_inventory import SmilesListInventory
 from syntheseus.search.node_evaluation import common as node_evaluation_common
 from syntheseus.search.utils.misc import lookup_by_name
-from syntheseus.search.visualization import visualize_andor, visualize_molset
+
+try:
+    # Try to import the visualization code, which will work only if `graphviz` is installed.
+    from syntheseus.search.visualization import visualize_andor, visualize_molset
+except ModuleNotFoundError:
```

Comment:
is there additional reporting we want to do here, e.g. log/print that this error was excepted?

Reply:
One idea would be:

```python
try:
    # Try to import the visualization code, which will work only if `graphviz` is installed.
    from syntheseus.search.visualization import visualize_andor, visualize_molset
    VIZ_CODE_IMPORTED = True
except ModuleNotFoundError:
    VIZ_CODE_IMPORTED = False
```

Then, in the main CLI somewhere, maybe write:

```python
assert VIZ_CODE_IMPORTED or config.num_routes_to_plot == 0
```